packages/@react-aria/selection/src/utils.ts

```diff
@@ -22,3 +24,7 @@ export function isNonContiguousSelectionModifier(e: Event) {
   // On Windows and Ubuntu, Alt + Space has a system wide meaning.
   return isAppleDevice() ? e.altKey : e.ctrlKey;
 }
+
+export function isSelectionAddition(e: LongPressEvent | PressEvent | PointerEvent) {
```

Comment:
I meant move the `isCtrlKeyPressed` function that is in `useSelectableItem.ts` into here.

Reply:
isCtrlKeyPressed was existing, but should be be more specific and include not windows. isMetaKeyOnWindowsPressed was new. My original refactor had both of these as functions. However, these two and the touch or pointer events are really getting at the same functionality of adding/removing rows by these events. Which is why there is one function that does all three.

Reply:
a) It shouldn't be windows specific, but rather non-mac specific. Linux also uses the Ctrl key and not the Command key. The logic in `isCtrlKeyPressed` is correct in that regard.
b) My suggestion was to reuse and centralize the existing code so we only have this check in one place. I guess this is also a fine function to have if it used `isCtrlKeyPressed`, but it is only used in a single place so it would also be fine to inline the touch and virtual part of that inside `useSelectableItem` like you had before and only share the part that is actually shared.